src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -10,6 +10,22 @@
 
 namespace Microsoft.VisualStudio.Services.Agent
 {
+    public enum SignatureVerificationMode
+    {
+        Error,
+        Warn,
```

Comment:
nit: "Warn" is the only verb in the list. Warning might be more consistent with Error and None.

Reply:
I was actually thinking of Error as a verb too, but None is still inconsistent. I'm ok with switching to Warning.